homeassistant/components/sensor/nest.py

```diff
@@ -129,6 +143,8 @@ def update(self):
 
         if self.variable == 'operation_mode':
             self._state = getattr(self.device, "mode")
+        elif self.variable == 'eta':
+            self._state = getattr(self.device, 'eta_begin')
```

Comment:
I know that you are just following the existing code, but maybe these string should be moved to constants 

at least harmonize the use of ' and "

Reply:
Yes, a dictionary should be used here:

```python
SENSOR_VARS = {'operation_mode': 'mode',
               'eta': 'eta_begin'}
```